### ISSUE TITLE
Application Insights Status Monitor requires TLS 1.0 to sign-in to Azure

### ISSUE THREAD

**Issue by seanksullivan** [microsoft/ApplicationInsights-Home#221]
Application Insights Status Monitor requires TLS 1.0 to sign-in to Azure.  This sign-in functionality should work with TLS 1.2 communications.

**Comment by MS-TimothyMothra** [comment]
**Workaround**

Please use the [PowerShell install instructions](https://docs.microsoft.com/azure/azure-monitor/app/monitor-performance-live-website-now#automation-with-powershell)

```powershell
Import-Module 'C:\Program Files\Microsoft Application Insights\Status Monitor\PowerShell\Microsoft.Diagnostics.Agent.StatusMonitor.PowerShell.dll'
Start-ApplicationInsightsMonitoring -Name appName -InstrumentationKey 00000000-000-000-000-0000000
```

----

**What's going on here**

Status Monitor attempts to log into Azure to retrieve your resources and ikey.
Status Monitor does not work for TLS 1.2. 

Microsoft's guidance is that an application should not set the TLS version and allow the OS to select the highest supported version. https://docs.microsoft.com/en-us/dotnet/framework/network-programming/tls#audit-your-code-and-make-code-changes

Unfortunately, 
- Status Monitor was build on .NET 4.5 which does not use the OS default.
- Status Monitor explicitly uses TLS 1.0 


----

**Looking Forward**

We are working on a replacement for StatusMonitor that will address this and other issues.
I don't have a release date at this time.

**Comment by MS-TimothyMothra** [comment]
@seanksullivan 
I’ve just published v0.1-alpha of IISConfigurator. This is our replacement for StatusMonitor.
**This does solve the TLS bug!**

I’ve organized all our docs for the IISConfigurator here:  https://github.com/Microsoft/ApplicationInsights-Home/blob/master/Samples/IISConfigurator/TOC.md
This includes:
	- A Quick Start Guide
	- Detailed getting started instructions
	- API reference for the new PowerShell Module
	- A page dedicated to Troubleshooting and Known Issues


Disclaimer: This is a prototype application. We do not recommend using this on your production environments.
Please pay close attention to the known issues.
Your apps may have issues if they include any of the following DLLs in the bin directory:
	· Microsoft.ApplicationInsights.dll
	· Microsoft.AspNet.TelemetryCorrelation.dll
	· System.Diagnostics.DiagnosticSource.dll
 

I am very interested in your feedback!
Please don’t hesitate to contact me if you have any issues or comments.
Specifically,
	- I would love to hear about your experiences getting started
	- And what you would like to see moving forward

**Comment by MS-TimothyMothra** [comment]
Today I've released the Release Candidate for Status Monitor v2.
Because this eliminates the TLS bug, I'm going to close this issue.

Please review or documentation to get started: https://docs.microsoft.com/en-us/azure/azure-monitor/app/status-monitor-v2-overview
